src/enemy/bowMoblin.c

```diff
@@ -1,37 +1,384 @@
+#define NENT_DEPRECATED
 #include "global.h"
 #include "entity.h"
 #include "enemy.h"
 #include "functions.h"
 
-extern void sub_0803C5F0(Entity*);
+typedef struct {
+    Entity base;
+    u8 filler[0x12];
+    u8 unk_0x7a;
+    u8 unk_0x7b;
+    u8 filter[0x4];
+    u8 unk_0x80;
+    u8 unk_0x81;
+    u8 unk_0x82;
+    u8 unk_0x83;
+} BowMoblinEntity;
```

Comment:
I think we've been padding entity structs to always be 0x88 bytes in size (At least I've been doing that).

Reply:
I think it's fine not to pad, the specialized types are never directly allocated